src/java.desktop/share/classes/javax/swing/plaf/synth/SynthTabbedPaneUI.java

```diff
@@ -647,12 +637,6 @@ private void paintTab(SynthContext ss, Graphics g,
             }
         }
 
-        if (isSelected) {
-            g.setColor(selectColor);
-        } else {
-            g.setColor(getUnselectedBackgroundAt(tabIndex));
-        }
-
         tabContext.getPainter().paintTabbedPaneTabBackground(tabContext, g,
```

Comment:
and this
if (tabsOpaque || tabPane.isOpaque()) {

Reply:
same reason as above.